twolde.py

```diff
@@ -205,11 +207,18 @@ def nice_time(seconds):
 
 def main():
     """ main """
-    parser = argparse.ArgumentParser(description='')
+    parser = argparse.ArgumentParser(
+        description="It's like Twitshift, but for people who prefer Twolde",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
     parser.add_argument(
         dest='command',
         choices=['install', 'run', 'remove'],
         help='the twolde command to run')
+    parser.add_argument(
```

Comment:
Yeah, you could just ignore it I suppose. I just tried it again though and it's nice that you can add help text for each command when you do it that way though.

```
usage: twolde.py [-h] {install,remove,run} ...

It's like Twitshift, but for people who prefer Twolde

positional arguments:
  {install,remove,run}  The Twolde command to run
    install             Install Twolde
    remove              Delete the Twolde config file at ~/.twolde/config.ini
    run                 Run Twolde

optional arguments:
  -h, --help            show this help message and exit
```